Cython/TestUtils.py

```diff
@@ -189,28 +190,25 @@ def unpack_source_tree(tree_file, dir=None):
         dir = tempfile.mkdtemp()
     header = []
     cur_file = None
-    f = open(tree_file)
-    try:
+    with open(tree_file, mode='rb') as f:
         lines = f.readlines()
-    finally:
-        f.close()
     del f
```

Comment:
This line probably needs to go as well...

Reply:
Right.